modules/apps/document-library/document-library-preview-pdf/src/main/java/com/liferay/document/library/preview/pdf/internal/configuration/admin/service/PDFPreviewManagedServiceFactory.java

```diff
@@ -41,9 +30,7 @@
 @Component(
 	configurationPid = "com.liferay.document.library.preview.pdf.internal.configuration.PDFPreviewConfiguration",
```

Comment:
Hey @RenanUmV, move the `configurationPid = "com.liferay.document.library.preview.pdf.internal.configuration.PDFPreviewConfiguration"` to the helper class as well. Thx